nengo/dists.py

```diff
@@ -294,6 +294,25 @@ def cdf(self, x):
             sq_x < 1., betainc(self.m / 2.0, self.n / 2.0, sq_x),
             np.ones_like(x))
 
+    def invcdf(self, y):
+        """Inverse cumulative distribution function.
+
+        Requires Scipy.
+
+        Parameters
+        ----------
+        y : ndarray
+            Cumulative probabilities in [0, 1].
+
+        Returns
+        -------
+        ndarray
+            Evaluation points `x` in [0, 1] such that `P(X <= x) = y`.
+        """
+        from scipy.special import betaincinv
```

Comment:
Usually if we have a SciPy dependency, we throw a special error if it's not found. Should this be done here as well?